build.gradle

```diff
@@ -92,3 +94,20 @@ task integrationTest(type: Test) {
 }
 
 check.dependsOn integrationTest
+
+spotbugsTest.enabled = false
```

Comment:
what's the reason for disabling this?

Reply:
I disabled this because I saw it was disabled in the [GP2GP](https://github.com/nhsconnect/integration-adaptor-gp2gp/blob/main/service/build.gradle) project. 

I'm not sure why it was disabled, but I agree with it as I don't think we should be analysing the test code for bugs, but am happy to enable it if others disagree.

Reply:
I'm fine with it being disabled for both test and intTest, personally.

Reply:
I was just curious as checkstyle analyses test code too. But i'm happy to be consistent with GP2GP